source/docs/installation/installation-frc.rst

```diff
@@ -0,0 +1,51 @@
+:orphan:
```

Comment:
We don't want to orphan these; for one thing, they don't show up in the generated PDF.